CHANGELOG.md

```diff
@@ -31,6 +31,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Add a pair filter strategy as an alternative to linesearch methods
 - Add a python example of a locomotion OCP with the robot Talos
+- Add two nonlinear centroidal dynamics whose control is respectively contact forces and derivative of contact forces
```

Comment:
```suggestion
- Add two nonlinear centroidal dynamical models, where the control is respectively contact forces and their time derivative
```

Reply:
Moreover these additions should be under "Unreleased". Don't know how the previous changes since the last release got added under 0.4.0 but that was a mistake